src/core/ext/client_config/lb_policy_factory.h

```diff
@@ -47,8 +47,19 @@ struct grpc_lb_policy_factory {
   const grpc_lb_policy_factory_vtable *vtable;
 };
 
+/** A resolved address alongside any LB related information associated with it.
+ * \a user_data, if not NULL, contains opaque data meant to be consumed by the
+ * gRPC LB policy. Note that no all LB policies support \a user_data as input.
+ * Those who don't will simply ignore it and will correspondingly return NULL in
```

Comment:
WRT your last paragraph, that was my initial idea too. After talking to @ctiller , it turned out it's not that easy: there's no easy way to get an address out of the connected subchannel instance the RR policy would provide. There are ways to do so once there's a call going over said subchannel, but not before that.
I agree it'd be wonderful to be able to do this. @ctiller , is there any fundamental reason why we wouldn't wanna expose the target of a connected subchannel?